README.md

```diff
@@ -38,11 +38,53 @@ main = runTwilio' (getEnv "ACCOUNT_SID")
   liftIO $ print calls
 
   -- Send a Message.
-  let body = PostMessage "+14158059869" "+14158059869" "Oh, hai"
+  let receivingPhone = "+14158059869"
+  let sendingPhone = "+14158059869"
+  let body = PostMessage receivingPhone sendingPhone "Oh, hai"
   message <- post body
   liftIO $ print message
 ```
 
+Example 2 - using SID and secret embedded in the code
+-------
+
+You can create a REST API client and send a message as follows:
+
+```hs
+{-#LANGUAGE OverloadedStrings #-}
+
+module Main where
+import Twilio.Types.SID
+import Control.Monad.IO.Class (liftIO)
+import Twilio
+import Twilio.Messages
+import qualified Data.Text as T
+
+parseCredentials :: T.Text -> T.Text -> Maybe Credentials
+parseCredentials accountSID authToken =
+    case parseAuthToken authToken of
+        Just authToken ->
+            (case parseSID accountSID of
+                Just accountSID ->
+                    Just (accountSID, authToken)
+                Nothing -> Nothing)
+        Nothing -> Nothing
```

Comment:
Here you are pattern matching on the success/failure of the parses to determine the next step. Instead of explicit pattern matching, we can rely on the behavior of Maybe's Functor and Applicative instances to write this more succinctly. For example,

```hs
parseCredentials :: Text -> Text -> Maybe Credentials
parseCredentials accountSID authToken =
  (,) <$> parseAccountSID accountSID <*> parseAuthToken authToken
```

I wonder if such a function should be include the library proper.